e2e_test/streaming/with_version_column.slt

```diff
@@ -0,0 +1,88 @@
+
+statement ok
+SET RW_IMPLICIT_FLUSH TO true;
+
+statement error
```

Comment:
could you also add a test for `on conflict IGNORE` and `with version column`? I think there should be an error for it.